libraries/core-react/src/Accordion/AccordionItem.tsx

```diff
@@ -16,7 +24,7 @@ const AccordionItem = forwardRef(function AccordionItem(
   },
   ref,
 ) {
-  const [expanded, setExpanded] = useState(isExpanded)
+  const [expanded, setExpanded] = useState<boolean>(isExpanded)
```

Comment:
I wasn't sure if we should add <boolean> explicit or not since Typescript used type inference correct and eslint was  happy 🤷‍♀️  But maybe it is a good idea to write it explicit anyway?

Reply:
Probably something that snuck in when I was refactoring. It was inferred correctly when I removed it 👍